var/spack/repos/builtin/packages/fpm/package.py

```diff
@@ -51,7 +54,11 @@ def install(self, spec, prefix):
         This functionality is provided by the ``install.sh`` script.
         """
 
-        script = Executable("./install.sh")
+        # Perform `chmod +x ./install.sh`
+        script_path = './install.sh'
+        chmod(script_path, S_IXUSR)
```

Comment:
Do you care if the only permission on the script is execute?  Otherwise, based on the comment, I believe you want to do the following (assuming just importing `os` and `stat`):

```
st = os.stat(script_path)
os.chmod(script_path, st.st_mode | stat.S_IXUSR)
```

Reply:
And if you don't care, then my nitpicky suggestion would be to change your comment at line 57 :grin:.

Reply:
Good idea, lemme do that.